### PR TITLE
Nic's HW08

### PR DESCRIPTION
**Pull request by Nic-Mon**
unresolved indentation problem with part(d)

### CODE REVIEW

**Comment by Nic-Mon** [comment]
fixed part(d)

File: mimsmind1.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import random
```
Comment (danielsgriffin) [inline mimsmind1.py]: Check your cow counting vs. the example:
Example
```
Let's play the mimsmind1 game. You have 11 guesses.

Guess a 3-digit number: 789

0 bull(s), 0 cow(s). Try again: 891

0 bull(s), 0 cow(s). Try again: 189

1 bull(s), 0 cow(s). Try again: 135

1 bull(s), 1 cow(s). Try again: 111

1 bull(s), 1 cow(s). Try again: 132

1 bull(s), 1 cow(s). Try again: 125

2 bull(s), 0 cow(s). Try again: 123

Congratulations. You guessed the correct number in 8 tries
```
You:
```
Let's play the mimsmind1 game. You have 11 guesses.

Guess a 3-digit number: 789

0 bull(s), 0 cow(s). Try again: 891

0 bull(s), 0 cow(s). Try again: 189

1 bull(s), 0 cow(s). Try again: 135

1 bull(s), 1 cow(s). Try again: 111

1 bull(s), 1 cow(s). Try again: 132

1 bull(s), 1 cow(s). Try again: 125

2 bull(s), 0 cow(s). Try again: 123

Congratulations. You guessed the correct number in 8 tries
```

File: mimsmind1.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import sys
+
+def guess_game(length):
+
+	num = str(random.randint(0, (10**int(length))-1))
```
Comment (danielsgriffin) [inline mimsmind1.py]: ~10% of the time a 3-digit number will be lower than 100. What happens then?

**Comment by danielsgriffin** [comment]
Good job. A couple testing issues.

**Pull request closed without merging**